<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>depositWindow</class>
 <widget class="QMainWindow" name="depositWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Сумма вкллада</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Срок (месяц)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Процентная ставка </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>186</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Налоговая ставка</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_cap">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Капитализация ежемесячно</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>331</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Пополнение ежемесячно</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_sum_dep">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="maximum">
     <double>10000000000.000000000000000</double>
    </property>
    <property name="value">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_month">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <double>12.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_interest">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <double>8.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_tax_rate">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="specialValueText">
     <string>13</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>453</x>
      <y>20</y>
      <width>20</width>
      <height>571</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>487</x>
      <y>35</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border-radius: 15px;
    border: 1px solid #000000;
    background-color: #f6f6f6;
    alignment: AlignRight; /* Правильный способ установки выравнивания */
	color: black;
}</string>
    </property>
    <property name="text">
     <string>                 Сумма на вкладе к концу срока</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_total_sum">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>110</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border-radius: 15px;
    border: 1px solid #000000;
    background-color: #f6f6f6;
    alignment: AlignRight; /* Правильный способ установки выравнивания */
	color: black;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>195</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border-radius: 15px;
    border: 1px solid #000000;
    background-color: #f6f6f6;
    alignment: AlignRight; /* Правильный способ установки выравнивания */
	color: black;
}</string>
    </property>
    <property name="text">
     <string>Начисленные проценты</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>255</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border-radius: 15px;
    border: 1px solid #000000;
    background-color: #f6f6f6;
    alignment: AlignRight; /* Правильный способ установки выравнивания */
	color: black;
}</string>
    </property>
    <property name="text">
     <string>Сумма налога</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_interst_total">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>200</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border-radius: 15px;
    border: 1px solid #000000;
    background-color: #f6f6f6;
    alignment: AlignRight; /* Правильный способ установки выравнивания */
	color: black;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>260</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border-radius: 15px;
    border: 1px solid #000000;
    background-color: #f6f6f6;
    alignment: AlignRight; /* Правильный способ установки выравнивания */
	color: black;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>331</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Расчитать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>420</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_add">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>331</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <double>10000000000.000000000000000</double>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
